JASP-Desktop/html/css/lightTheme-jasp.css

```diff
@@ -650,3 +650,10 @@ pre {
 ::-webkit-scrollbar-thumb:hover {
  box-shadow: inset 0 0 0 10px #97C4F2;
 }
+
+
+/* Custom values */
+.ql-container.ql-snow {
```

Comment:
Shouldnt this be in darkTheme as well?

Reply:
For dark theme, the default color already looks good. So I didn't change it

Reply:
So this isn't related to the weird onhover color in darkTheme?